nucypher/cli/commands/worklock.py

```diff
@@ -126,15 +126,17 @@ def bid(general_config, worklock_options, registry_options, force, hw_wallet, va
                                                                 poa=registry_options.poa,
                                                                 network=registry_options.network,
                                                                 show_balances=True)
+
+    registry = registry_options.get_registry(emitter, general_config.debug)
+    bidder = worklock_options.create_bidder(registry=registry, hw_wallet=hw_wallet)
+
     if not value:
         if force:
             raise click.MissingParameter("Missing --value.")
-        value = click.prompt("Enter bid amount in ETH", type=click.STRING)
+        minimum_bid = bidder.worklock_agent.minimum_allowed_bid
+        value = click.prompt(f"Enter bid amount in ETH (at least {Web3.fromWei(minimum_bid, 'ether')})", type=click.STRING)
```

Comment:
Is the type `String` intended here? vs Float? Could help cleanup any errors that occur on line 138 - eg. descriptive error vs exception perhaps?

Reply:
Changed to Float